Dockerfile

```diff
@@ -29,25 +28,23 @@ ARG DEBIAN_FRONTEND="noninteractive"
 ARG DEBCONF_NOWARNINGS="yes"
 
 # install system dependencies
-ARG SYS_DEPS="\
-    git \
-    libcurl4-openssl-dev \
-    libssl-dev \
-    openssh-client \
-    wget \
-    "
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends $SYS_DEPS \
+    && apt-get install -y --no-install-recommends \
+      git=1:2.34.* \
+      libcurl4-openssl-dev=7.81.* \
+      libicu-dev=70.* \
+      libssl-dev=3.0.* \
+      openssh-client=1:8.* \
+      pandoc=2.9.2.* \
```

Comment:
I would strongly prefer not installing pandoc because it's a significant complication (and size) that's totally unnecessary for this Docker image. My idea was that if one made a new repo/workflow that needed to do reporting, it would start from this as a base and install pandoc if it's necessary. That puts the dependencies close (in the same repo) to the things that actually depend on it.

Reply:
I would also prefer to not install pandoc, but until we resolve https://github.com/RMI-PACTA/pacta.portfolio.utils/issues/16, it is a system dependency for this set of packages, and we should reflect that.

Reply:
schade, I did not realize pandoc was a hard dependency of rmarkdown
https://github.com/rstudio/rmarkdown/blob/e62b928c5fea1b176df9d67704562af7f0b6e6cb/DESCRIPTION#L92

yeah, we should strip that out of pacta.portfolio.utils

Reply:
I opened #21 so that we don't forget to scrub it out, but I'm inclined to keep it here for now